### PR TITLE
[MI-869] Render link to shopify order page

### PR DESCRIPTION
**Pull request by samgavinio**
/cc @zendesk/mintegrations

### Description

Adds hyperlink to redirect to Shopify order page from the app.

screen shot 2016-09-26 at 5 52 19 pm

### References
* JIRA: https://zendesk.atlassian.net/browse/MI-869

### Risks
* [low] Hyperlink may not redirect correctly to shopify

### CODE REVIEW

File: src/templates/partials/order.hdbs [hunk 1]
```diff
@@ -57,6 +57,11 @@
         <div class="order-note">{{note}}</div>
       </div>
       {{/if}}
+      {{#if uri}}
+      <div class="u-mv-sm">
```
Comment (miogalang) [inline src/templates/partials/order.hdbs]: Should we not indent the contents of the {{if}} to make it easier to read?

**Comment by samgavinio** [comment]
💇 @miogalang

File: src/translations/en.yml [hunk 2]
```diff
@@ -220,7 +220,7 @@ parts:
 - translation:
       key: "txt.apps.shopify_app.orders.view"
       title: "Link text to show the order on Shopify"
-      value: "View order"
+      value: "Click here to view order in Shopify"
```
Comment (joseconsador) [inline src/translations/en.yml]: @samgavinio I'm not sure if we can change this string directly based on this page https://zendesk.atlassian.net/wiki/display/i18n/String+Modification+and+Deletion . 

@pdeuter 
This repo is already being watched by the internationalization team, but the location of the yml file will change, can we keep the old one or proceed like changes are updates to the translations?

**Comment by samgavinio** [comment]
💇 @joseconsador used a new string for the semantical string change.

**Comment by joseconsador** [comment]
👍

**Comment by jmramos02** [comment]
👍

**Pull request merged**